src/Condition.js

```diff
@@ -144,15 +153,22 @@ const propertyConditionMap = {
     allowed_ops: '=~',
   },
   url_param: {
-    express: (req, name) => {
+    vcl: (property) => {
+      let vcl = `subfield(req.url.qs, "${property.name}", "&")`;
```

Comment:
eslint may complain that the inner `vcl` can be confused with the outer `vcl`. And I'd add that it would be better and clearer to use `const` instead of `let` and just return from within the `if`.

Reply:
Actually, it doesn't complain. Anyway, I'll drop out of the if-clause as you suggested.